### PR TITLE
Use fedora 41 instead of rawhide

### PR DESCRIPTION
**Pull request by tssurya**
We are running into issues with our CI:
```
13.52 >>> Librepo error: Cannot download repomd.xml: Cannot download repodata/repomd.xFailed to download metadata (metalink: "https://mirrors.fedoraproject.org/metalink?repo=rawhide&arch=x86_64") for repository "rawhide"
13.52  Librepo error: Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
------
```

@dceara helped us get unblocked by doing a build: https://koji.fedoraproject.org/koji/taskinfo?taskID=123480682 on fedora41 to make this rpm available, let's use that meanwhile waiting for https://bodhi.fedoraproject.org/updates/FEDORA-2024-91bf950011 to get karmas

### CODE REVIEW

**Comment by tssurya** [comment]
https://pagure.io/fedora-infrastructure/issue/12183 blocked on this :(

**Comment by tssurya** [comment]
@JacobTanenbaum thanks! I need an lgtm on the GUI

**Review by dceara — APPROVED** [review]
Looks good to me, thanks!

**Comment by numansiddique** [comment]
FYI - https://bodhi.fedoraproject.org/updates/FEDORA-2024-a25161563d

and https://github.com/ovn-org/ovn-kubernetes/pull/4730

**Pull request merged**
